app/assets/javascripts/discourse/routes/build-category-route.js.es6

```diff
@@ -39,6 +39,12 @@ export default (filterArg, params) => {
             .filter(x => x)
             .join("/");
         }
+      } else if (Discourse.SiteSettings.slug_generation_method === "encoded") {
+        let separator = "/";
```

Comment:
I do not think it is required to define a variable for the separator. We hardcode `"/"` in so many places already.

```suggestion
```